### ISSUE TITLE
FSE- Cover block: use Featured image - image becomes blurred

### ISSUE THREAD

**Issue by paaljoachim** [WordPress/gutenberg#45577]
### Description

In the Site Editor using Twenty Twenty Three. In the Cover block.
Using a featured image the image was blurred on the frontend.
Getting the same image from the media library the frontend showed the image as sharp.

### Step-by-step reproduction instructions

1. In the Site Editor. (Using Twenty Twenty Three)
2. In the Single post template. Select to use Featured image.

![Screenshot 2022-11-06 at 23 45 02](https://user-images.githubusercontent.com/5323259/200199400-91900aa0-0c3f-420b-800d-f72cd122f846.jpg)

3. Check the frontend.

This is what I saw. Image is blurred.
![Screenshot 2022-11-06 at 23 46 31](https://user-images.githubusercontent.com/5323259/200199525-5fae171b-349f-453f-ba75-6c29bf44fe8d.jpg)

4. I went back and instead of using Featured image I went to the Media Library and got the image. 
This is what I saw on the frontend.

![Screenshot 2022-11-06 at 23 48 57](https://user-images.githubusercontent.com/5323259/200199609-39f8e4e1-882a-4c93-be39-214be81e63fd.jpg)

Getting the same image from the Media Library the frontend showed a sharp image. Using the same image as featured the image was blurred.

### Screenshots, screen recording, code snippet

-

### Environment info

WordPress 6.1
Gutenberg Version 14.4.0

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

No

**Comment by flexseth** [comment]
It could be possible that Auto Optimize plugin is attempting to optimize or resize the featured image below the resolution or size constraints that are needed for the cover block? 

Try disabling that plugin, and then 

1. regenerating thumbnails / clear cache
2. or uploading a completely new image to see if that resolves?

**Comment by paaljoachim** [comment]
Good thoughts @flexseth Seth!

Retesting. 
I deactivated all the plugins except Gutenberg.
Cleared the cache. 
Tested again with the featured image. The image is blurred on the frontend.
https://webkonsulenten.com/2021/02/25/a-snu-et-personlighetsmonster/

![Screenshot 2022-11-07 at 09 39 36](https://user-images.githubusercontent.com/5323259/200264421-1fc40a2c-60ba-4466-9f00-b4f3f1848af6.jpg)

Image information as seen in the Media Library.
File size: 177 KB
Dimensions: 1200 by 800 pixels

Went to the post and switched featured image. The result was just as blurred.
Featured image is setting this size as seen on the frontend.

`<img width="373" height="280" src="https://webkonsulenten.com/wp-content/uploads/rebekah-blocker-1iOhhlebMk0-unsplash-373x280.jpg" class="wp-block-cover__image-background wp-post-image" alt="rebekah-blocker-1iOhhlebMk0-unsplash" decoding="async" loading="lazy" data-object-fit="cover">`

I switched back the original featured image in the post.
Went to the Site Editor -> Single template -> Cover block - switched featured image to selecting image from the Media library. Saved and checked the frontend. The image is sharp again.
Uses this code:

`<img decoding="async" loading="lazy" width="1200" height="800" class="wp-block-cover__image-background wp-image-229" alt="peter-conlan-unsplash-jump" src="https://webkonsulenten.com/wp-content/uploads/peter-conlan-unsplash-jump.jpg" data-object-fit="cover" srcset="https://webkonsulenten.com/wp-content/uploads/peter-conlan-unsplash-jump.jpg 1200w, https://webkonsulenten.com/wp-content/uploads/peter-conlan-unsplash-jump-300x200.jpg 300w, https://webkonsulenten.com/wp-content/uploads/peter-conlan-unsplash-jump-1024x683.jpg 1024w, https://webkonsulenten.com/wp-content/uploads/peter-conlan-unsplash-jump-768x512.jpg 768w" sizes="(max-width: 1200px) 100vw, 1200px">`

------------

The frontend code varies depending on if the image used in from the Media Library or a Featured Image.

Ps I also deactivated the Gutenberg plugin to see if it made any difference. It did not.
This is a dev site so I can let the featured image just stay visible for a while.

**Comment by draganescu** [comment]
I can't repro this. The twenty twenty three, by default, I see it uses a featured image block in the single template. I understand you changed that with a cover block?

In the screenshots the image is blurry because the SRC points to a small version, but the question is why/how :)

**Comment by flexseth** [comment]
Paal, 

The first link in your reply, the image is being upsampled. The actual image size is around 350px but is being resized to over 700px. This is why it's blurry (350px is way too small for a cover and can't be scaled up to 2x without looking blurry in any situation).

Below is Chrome dev tools that shows this

<img width="504" alt="image" src="https://user-images.githubusercontent.com/3792502/200345723-56d2ed23-5a1c-473e-83b9-93c19291baaf.png">

Here's a link to that actual image so you can take a look, it shows the file size in the link and you will see the image is this size. 
[Your 350px cover image](https://webkonsulenten.com/wp-content/uploads/peter-conlan-unsplash-jump-373x280.jpg)

What I think your best bet would be is to find a completely new image that 1000 px and and clear, upload that as the featured image without any other plugins turned on. Gutenberg doesn't even need to be on as featured image support on cover blocks is now part of WordPress Core. 

Give it a try with a completely new image. 

Also, did you regenerate thumbnails after removing Auto Optimize? That could be the problem with the initial image. [Here's a blog post](https://kinsta.com/blog/regenerate-thumbnails/) on how to do that

Best/Seth

**Comment by paaljoachim** [comment]
It was my feeling that it might be hard to reproduce.

> The twenty twenty three, by default, I see it uses a featured image block in the single template. I understand you changed that with a cover block?

I removed the featured image block because I am not able to add text on top of it like I can in the Cover block. So I decided to instead use the Cover block with featured image and then add some paragraph text onto it.

![Screenshot 2022-11-07 at 16 19 23](https://user-images.githubusercontent.com/5323259/200346838-730d0feb-f18b-46f3-bdfe-a712481e59ba.jpg)

> In the screenshots the image is blurry because the SRC points to a small version, but the question is why/how :)

Yes. That is the question.... Something to know about. Wonder about and see what ideas that pop up that it might point to.

**Comment by flexseth** [comment]
> I can't repro this. The twenty twenty three, by default, I see it uses a featured image block in the single template. I understand you changed that with a cover block?
> 
> In the screenshots the image is blurry because the SRC points to a small version, but the question is why/how :)

My guess is that Auto Optimize has resized this image somehow and he needs to regenerate thumbnails, perhaps and/or even has deleted the original file and left only the smaller version?

**Comment by paaljoachim** [comment]
I went ahead and installed the Regenerate Thumbnails plugin to see if that helped. Went through the process of regenerating the images.

I went ahead and made another post with a featured image.
rebekah-blocker-1iOhhlebMk0-unsplash.jpg
March 5, 2021
83 KB
1400 by 397 pixels

The same problem is seen.
https://webkonsulenten.com/2022/11/07/a-test-post/

Hey Seth. I am wondering if it might be as you say Auto Optimize causing an issue here, as I downloaded the original image, renamed it and uploaded it and set it as featured in the post. Checked the post on the frontend and noticed that it now was back to normal. I will need to alert the author of Auto Optimize that it seems in some way that the plugin is affecting the featured image feature inside the Cover block.

---
EDIT.
Added this Auto Optimize support issue: https://wordpress.org/support/topic/auto-optimize-plugin-affecting-featured-image-feature-in-cover-block/
